spring-integration-test/src/main/java/org/springframework/integration/test/mock/MockIntegration.java

```diff
@@ -85,12 +87,11 @@
 	 * @return the mocked {@link MessageSource}
 	 * @see Mockito#mock(Class)
 	 */
-	@SuppressWarnings("rawtypes")
 	public static MessageSource<?> mockMessageSource(Message<?> message) {
 		MessageSource messageSource = Mockito.mock(MessageSource.class);
 
-		given(messageSource.receive())
-				.<Message<?>>willReturn(message);
+		BDDMockito.given(messageSource.receive())
```

Comment:
Right, we prefer but in `test` code.
This one is in `main`, so production code. Therefore no static imports.